2019-05-11T16:52:15,166 DEBUG io.netty.util.internal.logging.InternalLoggerFactory:71 - Using SLF4J as the default logging framework
2019-05-11T16:52:15,174 DEBUG io.netty.util.ResourceLeakDetector:81 - -Dio.netty.leakDetection.level: simple
2019-05-11T16:52:15,175 DEBUG io.netty.util.ResourceLeakDetector:81 - -Dio.netty.leakDetection.maxRecords: 4
2019-05-11T16:52:15,252 DEBUG io.netty.channel.MultithreadEventLoopGroup:76 - -Dio.netty.eventLoopThreads: 8
2019-05-11T16:52:15,330 DEBUG io.netty.util.internal.PlatformDependent:71 - Platform: Windows
2019-05-11T16:52:15,333 DEBUG io.netty.util.internal.PlatformDependent0:76 - -Dio.netty.noUnsafe: false
2019-05-11T16:52:15,334 DEBUG io.netty.util.internal.PlatformDependent0:76 - Java version: 8
2019-05-11T16:52:15,336 DEBUG io.netty.util.internal.PlatformDependent0:71 - sun.misc.Unsafe.theUnsafe: available
2019-05-11T16:52:15,337 DEBUG io.netty.util.internal.PlatformDependent0:71 - sun.misc.Unsafe.copyMemory: available
2019-05-11T16:52:15,337 DEBUG io.netty.util.internal.PlatformDependent0:71 - java.nio.Buffer.address: available
2019-05-11T16:52:15,338 DEBUG io.netty.util.internal.PlatformDependent0:71 - direct buffer constructor: available
2019-05-11T16:52:15,339 DEBUG io.netty.util.internal.PlatformDependent0:76 - java.nio.Bits.unaligned: available, true
2019-05-11T16:52:15,340 DEBUG io.netty.util.internal.PlatformDependent0:71 - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable prior to Java9
2019-05-11T16:52:15,340 DEBUG io.netty.util.internal.PlatformDependent0:76 - java.nio.DirectByteBuffer.<init>(long, int): available
2019-05-11T16:52:15,340 DEBUG io.netty.util.internal.PlatformDependent:76 - sun.misc.Unsafe: available
2019-05-11T16:52:15,341 DEBUG io.netty.util.internal.PlatformDependent:76 - -Dio.netty.tmpdir: C:\Users\jin\AppData\Local\Temp (java.io.tmpdir)
2019-05-11T16:52:15,342 DEBUG io.netty.util.internal.PlatformDependent:76 - -Dio.netty.bitMode: 64 (sun.arch.data.model)
2019-05-11T16:52:15,343 DEBUG io.netty.util.internal.PlatformDependent:76 - -Dio.netty.noPreferDirect: false
2019-05-11T16:52:15,343 DEBUG io.netty.util.internal.PlatformDependent:76 - -Dio.netty.maxDirectMemory: 1653604352 bytes
2019-05-11T16:52:15,346 DEBUG io.netty.util.internal.PlatformDependent:76 - -Dio.netty.uninitializedArrayAllocationThreshold: -1
2019-05-11T16:52:15,347 DEBUG io.netty.util.internal.CleanerJava6:71 - java.nio.ByteBuffer.cleaner(): available
2019-05-11T16:52:15,391 DEBUG io.netty.channel.nio.NioEventLoop:76 - -Dio.netty.noKeySetOptimization: false
2019-05-11T16:52:15,391 DEBUG io.netty.channel.nio.NioEventLoop:76 - -Dio.netty.selectorAutoRebuildThreshold: 512
2019-05-11T16:52:15,471 DEBUG io.netty.util.internal.PlatformDependent:71 - org.jctools-core.MpscChunkedArrayQueue: available
2019-05-11T16:52:16,172 DEBUG io.netty.resolver.dns.DefaultDnsServerAddressStreamProvider:76 - Default DNS servers: [/192.168.1.1:53] (sun.net.dns.ResolverConfiguration)
2019-05-11T16:52:16,180 DEBUG io.netty.util.NetUtil:76 - -Djava.net.preferIPv4Stack: false
2019-05-11T16:52:16,180 DEBUG io.netty.util.NetUtil:76 - -Djava.net.preferIPv6Addresses: false
2019-05-11T16:52:16,366 DEBUG io.netty.util.NetUtil:86 - Loopback interface: lo (Software Loopback Interface 1, 127.0.0.1)
2019-05-11T16:52:16,367 DEBUG io.netty.util.NetUtil:81 - Failed to get SOMAXCONN from sysctl and file \proc\sys\net\core\somaxconn. Default: 200
2019-05-11T16:52:16,476 INFO  com.mchange.v2.log.MLog:212 - MLog clients using slf4j logging.
2019-05-11T16:52:16,481 DEBUG com.mchange.v2.log.MLog:207 - Reading VM config for path list /com/mchange/v2/log/default-mchange-log.properties, /mchange-commons.properties, /c3p0.properties, hocon:/reference,/application,/c3p0,/, /mchange-log.properties, /
2019-05-11T16:52:16,481 DEBUG com.mchange.v2.log.MLog:207 - The configuration file for resource identifier '/mchange-commons.properties' could not be found. Skipping.
2019-05-11T16:52:16,482 DEBUG com.mchange.v2.log.MLog:207 - The configuration file for resource identifier '/c3p0.properties' could not be found. Skipping.
2019-05-11T16:52:16,482 DEBUG com.mchange.v2.log.MLog:207 - The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.
2019-05-11T16:52:16,483 DEBUG com.mchange.v2.log.MLog:207 - The configuration file for resource identifier '/mchange-log.properties' could not be found. Skipping.
2019-05-11T16:52:16,484 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/mchange-commons.properties' could not be found. Skipping.
2019-05-11T16:52:16,484 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/mchange-log.properties' could not be found. Skipping.
2019-05-11T16:52:16,485 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.
2019-05-11T16:52:16,485 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/c3p0.properties' could not be found. Skipping.
2019-05-11T16:52:16,526 INFO  com.mchange.v2.c3p0.C3P0Registry:212 - Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
2019-05-11T16:52:16,582 DEBUG com.mchange.v2.c3p0.management.DynamicPooledDataSourceManagerMBean:204 - MBean: com.mchange.v2.c3p0:type=PooledDataSource,identityToken=1hgf11da21a31zau8hzjor|7b5de786,name=1hgf11da21a31zau8hzjor|7b5de786 registered.
2019-05-11T16:52:16,603 DEBUG com.mchange.v2.c3p0.management.DynamicPooledDataSourceManagerMBean:204 - MBean: com.mchange.v2.c3p0:type=PooledDataSource,identityToken=1hgf11da21a31zau8hzjor|7b5de786,name=1hgf11da21a31zau8hzjor|7b5de786 unregistered, in order to be reregistered after update.
2019-05-11T16:52:16,603 DEBUG com.mchange.v2.c3p0.management.DynamicPooledDataSourceManagerMBean:204 - MBean: com.mchange.v2.c3p0:type=PooledDataSource,identityToken=1hgf11da21a31zau8hzjor|7b5de786,name=1hgf11da21a31zau8hzjor|7b5de786 registered.
2019-05-11T16:52:16,743 INFO  com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource:212 - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgf11da21a31zau8hzjor|7b5de786, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.cj.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgf11da21a31zau8hzjor|7b5de786, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost/tjsdk?characterEncoding=UTF-8&useSSL=false&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&autoReconnect=true  , maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 50, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2019-05-11T16:52:16,768 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/mchange-commons.properties' could not be found. Skipping.
2019-05-11T16:52:16,768 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/mchange-log.properties' could not be found. Skipping.
2019-05-11T16:52:16,769 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier '/c3p0.properties' could not be found. Skipping.
2019-05-11T16:52:16,769 DEBUG com.mchange.v2.cfg.MConfig:207 - The configuration file for resource identifier 'hocon:/reference,/application,/c3p0,/' could not be found. Skipping.
2019-05-11T16:52:16,770 DEBUG com.mchange.v2.resourcepool.BasicResourcePool:204 - com.mchange.v2.resourcepool.BasicResourcePool@536c109c config: [start -> 3; min -> 3; max -> 50; inc -> 3; num_acq_attempts -> 30; acq_attempt_delay -> 1000; check_idle_resources_delay -> 0; max_resource_age -> 0; max_idle_time -> 0; excess_max_idle_time -> 0; destroy_unreturned_resc_time -> 0; expiration_enforcement_delay -> 0; break_on_acquisition_failure -> false; debug_store_checkout_exceptions -> false; force_synchronous_checkins -> false]
2019-05-11T16:52:16,771 DEBUG com.mchange.v2.c3p0.impl.C3P0PooledConnectionPoolManager:204 - Created new pool for auth, username (masked): 'ro******'.
2019-05-11T16:52:16,771 DEBUG com.mchange.v2.resourcepool.BasicResourcePool:204 - acquire test -- pool size: 0; target_pool_size: 3; desired target? 1
2019-05-11T16:52:16,771 DEBUG com.mchange.v2.resourcepool.BasicResourcePool:204 - awaitAvailable(): [unknown]
2019-05-11T16:52:17,107 INFO  com.core.verticle.CooperationDatabaseVerticle:39 - database start success
2019-05-11T16:52:17,710 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.numHeapArenas: 8
2019-05-11T16:52:17,711 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.numDirectArenas: 8
2019-05-11T16:52:17,711 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.pageSize: 8192
2019-05-11T16:52:17,711 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.maxOrder: 11
2019-05-11T16:52:17,711 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.chunkSize: 16777216
2019-05-11T16:52:17,713 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.tinyCacheSize: 512
2019-05-11T16:52:17,714 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.smallCacheSize: 256
2019-05-11T16:52:17,714 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.normalCacheSize: 64
2019-05-11T16:52:17,714 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.maxCachedBufferCapacity: 32768
2019-05-11T16:52:17,714 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.cacheTrimInterval: 8192
2019-05-11T16:52:17,715 DEBUG io.netty.buffer.PooledByteBufAllocator:76 - -Dio.netty.allocator.useCacheForAllThreads: true
2019-05-11T16:52:17,720 DEBUG io.netty.util.internal.InternalThreadLocalMap:76 - -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024
2019-05-11T16:52:17,720 DEBUG io.netty.util.internal.InternalThreadLocalMap:76 - -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096
2019-05-11T16:52:17,775 DEBUG io.netty.resolver.dns.UnixResolverDnsServerAddressStreamProvider:86 - failed to parse /etc/resolv.conf and/or /etc/resolver
java.lang.IllegalArgumentException: \etc\resolv.conf didn't provide any name servers
	at io.netty.resolver.dns.UnixResolverDnsServerAddressStreamProvider.<init>(UnixResolverDnsServerAddressStreamProvider.java:98) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.dns.UnixResolverDnsServerAddressStreamProvider.<init>(UnixResolverDnsServerAddressStreamProvider.java:124) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.dns.UnixResolverDnsServerAddressStreamProvider.parseSilently(UnixResolverDnsServerAddressStreamProvider.java:66) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.dns.DnsServerAddressStreamProviders.<clinit>(DnsServerAddressStreamProviders.java:28) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.dns.DnsNameResolverBuilder.<init>(DnsNameResolverBuilder.java:54) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.vertx.core.impl.resolver.DnsResolverProvider$1$1.newNameResolver(DnsResolverProvider.java:137) ~[vertx-core-3.5.0.jar:?]
	at io.netty.resolver.dns.DnsAddressResolverGroup.newResolver(DnsAddressResolverGroup.java:86) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.dns.DnsAddressResolverGroup.newResolver(DnsAddressResolverGroup.java:73) ~[netty-resolver-dns-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.resolver.AddressResolverGroup.getResolver(AddressResolverGroup.java:69) ~[netty-resolver-4.1.15.Final.jar:4.1.15.Final]
	at io.vertx.core.impl.resolver.DnsResolverProvider$1.newResolver(DnsResolverProvider.java:184) ~[vertx-core-3.5.0.jar:?]
	at io.netty.resolver.AddressResolverGroup.getResolver(AddressResolverGroup.java:69) ~[netty-resolver-4.1.15.Final.jar:4.1.15.Final]
	at io.vertx.core.impl.AddressResolver.resolveHostname(AddressResolver.java:83) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.VertxImpl.resolveAddress(VertxImpl.java:766) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.net.impl.AsyncResolveConnectHelper.doBind(AsyncResolveConnectHelper.java:87) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.http.impl.HttpServerImpl.listen(HttpServerImpl.java:344) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.http.impl.HttpServerImpl.listen(HttpServerImpl.java:231) ~[vertx-core-3.5.0.jar:?]
	at com.core.verticle.CooperationHttpVerticle.start(CooperationHttpVerticle.java:74) ~[classes/:?]
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$10(DeploymentManager.java:481) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:344) ~[vertx-core-3.5.0.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [netty-common-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:403) [netty-common-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463) [netty-transport-4.1.15.Final.jar:4.1.15.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [netty-common-4.1.15.Final.jar:4.1.15.Final]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
2019-05-11T16:52:17,824 DEBUG io.netty.channel.DefaultChannelId:76 - -Dio.netty.processId: 11748 (auto-detected)
2019-05-11T16:52:18,102 DEBUG io.netty.channel.DefaultChannelId:76 - -Dio.netty.machineId: 00:50:56:ff:fe:c0:00:01 (auto-detected)
2019-05-11T16:52:18,122 DEBUG io.netty.buffer.ByteBufUtil:76 - -Dio.netty.allocator.type: pooled
2019-05-11T16:52:18,123 DEBUG io.netty.buffer.ByteBufUtil:76 - -Dio.netty.threadLocalDirectBufferSize: 65536
2019-05-11T16:52:18,131 DEBUG io.netty.buffer.ByteBufUtil:76 - -Dio.netty.maxThreadLocalCharBufferSize: 16384
2019-05-11T16:52:18,150 INFO  com.core.verticle.CooperationHttpVerticle:76 - HTTP server running on port 8092
2019-05-11T16:53:15,023 DEBUG io.netty.util.Recycler:76 - -Dio.netty.recycler.maxCapacityPerThread: 32768
2019-05-11T16:53:15,023 DEBUG io.netty.util.Recycler:76 - -Dio.netty.recycler.maxSharedCapacityFactor: 2
2019-05-11T16:53:15,023 DEBUG io.netty.util.Recycler:76 - -Dio.netty.recycler.linkCapacity: 16
2019-05-11T16:53:15,023 DEBUG io.netty.util.Recycler:76 - -Dio.netty.recycler.ratio: 8
2019-05-11T16:53:15,039 DEBUG io.netty.buffer.AbstractByteBuf:81 - -Dio.netty.buffer.bytebuf.checkAccessible: true
2019-05-11T16:53:15,041 DEBUG io.netty.util.ResourceLeakDetectorFactory:76 - Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@3a61e7bd
2019-05-11T16:53:15,559 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:54:17,805 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:54:52,881 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:54:57,031 DEBUG com.mchange.v2.c3p0.impl.NewPooledConnection:207 - com.mchange.v2.c3p0.impl.NewPooledConnection@315fd8c0 handling a throwable.
java.sql.SQLException: Field 'app_id' doesn't have a default value
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1826) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2034) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:1970) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:4984) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1955) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384) ~[c3p0-0.9.5.2.jar:0.9.5.2]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:91) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:31) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$executeDirect$2(JDBCClientImpl.java:145) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:126) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:88) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$getConnection$3(JDBCClientImpl.java:192) ~[vertx-jdbc-client-3.5.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_181]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
2019-05-11T16:54:57,034 DEBUG com.mchange.v2.sql.SqlUtils:207 - Attempted to convert SQLException to SQLException. Leaving it alone. [SQLState: HY000; errorCode: 1364]
java.sql.SQLException: Field 'app_id' doesn't have a default value
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1826) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2034) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:1970) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:4984) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1955) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384) ~[c3p0-0.9.5.2.jar:0.9.5.2]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:91) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:31) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$executeDirect$2(JDBCClientImpl.java:145) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:126) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:88) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$getConnection$3(JDBCClientImpl.java:192) ~[vertx-jdbc-client-3.5.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_181]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
2019-05-11T16:54:58,161 DEBUG com.mchange.v2.c3p0.impl.DefaultConnectionTester:207 - Testing a Connection in response to an Exception:
java.sql.SQLException: Field 'app_id' doesn't have a default value
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1826) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2034) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:1970) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:4984) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1955) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384) ~[c3p0-0.9.5.2.jar:0.9.5.2]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:91) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:31) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$executeDirect$2(JDBCClientImpl.java:145) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:126) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:88) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$getConnection$3(JDBCClientImpl.java:192) ~[vertx-jdbc-client-3.5.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_181]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
2019-05-11T16:54:58,162 ERROR com.core.service.imp.CooperationDatabaseServiceImp:60 - Database query error
java.sql.SQLException: Field 'app_id' doesn't have a default value
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1826) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2034) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:1970) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:4984) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mysql.cj.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1955) ~[mysql-connector-java-6.0.5.jar:6.0.5]
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384) ~[c3p0-0.9.5.2.jar:0.9.5.2]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:91) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.actions.JDBCUpdate.execute(JDBCUpdate.java:31) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$executeDirect$2(JDBCClientImpl.java:145) ~[vertx-jdbc-client-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:126) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:88) ~[vertx-core-3.5.0.jar:?]
	at io.vertx.ext.jdbc.impl.JDBCClientImpl.lambda$getConnection$3(JDBCClientImpl.java:192) ~[vertx-jdbc-client-3.5.0.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_181]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
2019-05-11T16:54:58,165 INFO  com.core.verticle.CooperationHttpVerticle:171 - 添加应用失败
2019-05-11T16:56:20,767 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:56:20,809 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:57:42,317 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:57:45,026 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:57:52,382 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:57:52,424 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:57:58,020 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:57:58,073 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:00,472 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:00,511 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:02,586 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:02,612 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:04,940 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:04,969 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:07,220 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:07,253 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:10,554 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:10,580 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:13,146 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:13,180 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:15,292 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:15,336 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:17,409 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:17,432 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:20,490 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:20,517 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:23,043 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:23,067 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:26,026 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:26,052 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:29,490 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:29,516 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:31,249 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:31,287 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:32,955 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:33,026 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:34,771 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:34,804 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:38,185 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:38,212 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:41,480 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:41,504 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:43,489 INFO  com.core.verticle.CooperationHttpVerticle:168 - 添加应用成功
2019-05-11T16:58:43,521 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T16:58:45,335 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:03:13,723 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:03:15,656 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:03:16,365 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:03:16,986 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:04:52,544 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:04:53,347 INFO  com.core.verticle.CooperationHttpVerticle:145 - 获取公司信息列表成功
2019-05-11T17:04:55,382 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
2019-05-11T17:11:19,909 INFO  com.core.verticle.CooperationHttpVerticle:218 - 获取应用信息列表成功
